HW03/Dmitry_Susha/dmitry_susha_hw_03_t_03.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'time'
+class Homework3
+  def task3(logs)
```

Comment:
I will give you advice on how to create a good solution, so try refactoring your code:
Split log
Select the necessary rows that match the condition
Go through them with an iterator and select the necessary data (I advise you to use a regular expression)
Make a difference calculation method (Time.parse is handy to use)
create a condition if the selected data exists, then pass it to the created method for calculating the difference in events